### PR TITLE
Moved to version 4.6.3

### PR DESCRIPTION
**Pull request by gbastien**
Hi,

we updated to version 4.6.3 and fixed MANIFEST.in to include file *.woff2

Could you please review/merge/release?  If merged, I can do a release if you give me pypi rights ;-)  My pypi user is gauthierbastien.

Thank you!

Gauthier

### CODE REVIEW

**Comment by gbastien** [comment]
Hi @ebrehault @jcbrand could you have a look at this?

Thank you!

Gauthier

File: README.rst [hunk 1]
```diff
@@ -55,7 +55,7 @@ Available CSS classes and values for the content attribute can be found in the `
 
 An example page is avaible in the addon, here is a screenshot:
 
-.. image:: http://plone.org/products/collective.fontawesome/screenshot
+.. image:: http://old.plone.org/products/collective.fontawesome/screenshot
```
Comment (gforcada) [inline README.rst]: @gbastien as probably old.plone.org will not be around at some point, maybe copying the image in the repo would be best?

File: collective/fontawesome/configure.zcml [hunk 2]
```diff
@@ -28,6 +28,7 @@
       provides="Products.GenericSetup.interfaces.EXTENSION"
       />
 
+  <!--include package=".example" /-->
```
Comment (gforcada) [inline collective/fontawesome/configure.zcml]: @gbastien debugging leftovers?

File: README.rst (hunk 1 above)
Comment (gbastien) [inline README.rst]: Hi @gforcada I added the image into the github repository, it should display correctly also on pypi.

File: collective/fontawesome/configure.zcml (hunk 2 above)
Comment (gbastien) [inline collective/fontawesome/configure.zcml]: @gforcada I left it there so it is easier for newbies to get the example working, what do you think, should I remove it?

Comment (gforcada) [inline collective/fontawesome/configure.zcml]: I'm in *general* against keeping commented code, but if it has a purpose maybe add a comment on top of it then

Comment (gbastien) [inline collective/fontawesome/configure.zcml]: Hi @gforcada I added a comment, thank you for reviewing

**Comment by gbastien** [comment]
Hi @gforcada @ebrehault @jcbrand I think this PR is ok to be merged now, what do you think?

If you merge, I can do the release if you give me pypi rights.

Thank you,

Gauthier

**Comment by gbastien** [comment]
Nobody? ;-)

**Comment by gbastien** [comment]
Great, thank you.

Anybody for a release or able to give me pypi rights? ;-)

Gauthier

**Comment by gbastien** [comment]
Little reminder @ebrehault @toutpt could you please give me pypi rights or grant pypi package Owner role to user "collective"?

Thank you!

Gauthier

**Comment by gbastien** [comment]
Thank to @ebrehault that gave me pypi rights, I could make a new release ;-)

https://pypi.python.org/pypi/collective.fontawesome/

**Pull request merged**